test/jdk/sun/security/provider/SeedGenerator/SeedGeneratorChoice.java

```diff
@@ -26,7 +26,7 @@
  * @bug 6998583 8141039
  * @summary NativeSeedGenerator is making 8192 byte read requests from
  *             entropy pool on each init.
- * @run main/othervm -Djava.security.egd=file:/dev/random SeedGeneratorChoice
```

Comment:
This test uses generateSeed and should be run with egd pointing to /dev/urandom to avoid reading from /dev/random

Reply:
Backing out change.

Reply:
Changed to /dev/urandom.